testing/tests/ui/error_file_path.stderr

```diff
@@ -0,0 +1,29 @@
+error: failed to parse template source
+ --> testing/templates/invalid_syntax.html:1:14
```

Comment:
Okay, final nit: there should be two space before the `-->` to match rustc output, this will also help align the errors in case both show up.

Reply:
Oh interesting. There are two whitespace characters. I tried adding more but it continues to display. I can enter the unicode realm to try to trick my way around if you want?

Reply:
No, never mind then.